modules/virt-collecting-data-about-vms.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-collecting-virt-data.adoc
+
+[id="virt-collecting-data-about-vms_{context}"]
+= Collecting data about virtual machines
+
+Collecting data about malfunctioning virtual machines (VMs) minimizes the time required to analyze and determine the root cause.
+
+.Prerequisites
+
+* Windows VMs:
+** Record the Windows patch update details for Red Hat Support.
+** Install the latest version of the VirtIO drivers. The VirtIO drivers include the QEMU guest agent.
+** If Remote Desktop Protocol (RDP) is enabled, try to connect to the VMs with RDP to determine whether there is a problem with the connection software.
+
+.Procedure
+
+. Collect `must-gather` data for VMs.
+. Collect screenshots of VMs before they start.
+. Collect memory dumps from VMs _before_ remediation attempts.
```

Comment:
Remove this item. We need to wait for https://issues.redhat.com/browse/CNV-12593 to land before adding this to the diagnostics playbook, I've updated https://issues.redhat.com/browse/CNV-20256 to inclde the playbook update as well

Reply:
I'll comment out the line and mention the dependency. Thanks for adding the "update diagnostics playbook" sentence. I'll add the module name so that the writer knows which file needs to be changed.